solver:
  rollouts: 20
  lambda: 0.0001 #0.001
  h: 10.0
  step_size: 0.01
  horizon: 1.00

  cost_ratio: 0.2
  caching_factor: 0.3
  filtering: false
  substeps: 1

  gradient_step_size: 0.15
  momentum_step_size: 0.8

  filter_type: 1
  filter_order: 1
  filter_window: 5

  adaptive_sampling: false
  input_variance: [0.25, 0.25, 0.25, 0.25, 0.25, 0.25, 0.25, 0.05, 0.05, 0.05]

  discount_factor: 1.0
  verbose: false
  threads: 2
  debug_print: false

  use_tree_search: false
  pruning_threshold: 0.01

  logging: true

#  expert_types: {NORM, IMP}
  expert_weights: [0.1, 1.0]
  log_data: true


initial_configuration: [0.0, -0.52, 0.0, -1.785, 0.0, 1.10, 0.69]
linear_weight: 10.0
angular_weight: 1.0
obstacle_radius: 0.15

policy_update_rate: 0.0  # 0.0 execute as fast as possible
reference_update_rate: 10.0
publish_ros: true
ros_publish_rate: 10.0

sequential: true
static_optimization: false
sim_dt: 0.01
